dev/com.ibm.ws.logging/src/com/ibm/ws/logging/internal/impl/BaseTraceService.java

```diff
@@ -1658,18 +1661,23 @@ protected void captureSystemStreams() {
      * when the special trace components are created.
      */
     protected void restoreSystemStreams() {
-        if (System.out == teeOut)
-            System.setOut(systemOut.getOriginalStream());
-        if (System.err == teeErr)
-            System.setErr(systemErr.getOriginalStream());
+        /*
+         * OL17768 - No obvious cause of OL17768.
+         * Disable this check and "restore"
+         * it regardless to evaluate effects.
+         */
+        //if (System.out == teeOut)
```

Comment:
The if statements are for efficiency purpose only.  It doesn't change the logic.  I'm not sure this change will have any effect to the issue.

Reply:
Right, the cause of the issue seems to be that the `System.out` is not set back to the _original_ `System.out`. It fails since the check for `System.out` shows that the reference is pointing to a `TeePrintStream` (i.e. `teeOut`).

This check is done before each Junit execution. After each Junit execution it's _supposed_ to clean up (i.e. restore the System.out to the original stream). The Junits are suppose to execute sequentially so there isn't any obvious cause of issue here. This seems like it's the only possible area where it _may_ go wrong. As I am unable to reproduce  the issue at all, this PR is an attempt to evaluate if this could cause any change (if any). As this is just replacing the the reference (and only would be really hit through junits as runtime we would not expect the BaseTraceService to be shutting down during runtime, this would not impact runtime at all.